autogluon/training/docker/0.8/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,68 @@
+ARG PYTHON_VERSION=3.9.16
+
+FROM 763104351884.dkr.ecr.us-west-2.amazonaws.com/pytorch-training:1.13.1-cpu-py39-ubuntu20.04-sagemaker
+
+# PYTORCH_SKIP_CUDNN_COMPATIBILITY_CHECK - see https://github.com/autogluon/autogluon/issues/2534
+# ENV PYTORCH_SKIP_CUDNN_COMPATIBILITY_CHECK=1
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+# Install dependencies
+RUN apt-get update \
+ && apt-get -y upgrade \
+ && apt-get install -y --no-install-recommends \
+ && apt-get autoremove -y \
+ # Install tesseract-ocr to support automm document classification
+ && apt-get install tesseract-ocr -y \
+ # Install rsync to support ray distributed training
+ && apt-get install rsync -y \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+ARG AUTOGLUON_VERSION=0.8.2
+
+# NOTE: reinstalling horovod with pytorch support because it is available in the container
+# Update requests >= 2.31.0: CVE-2023-32681
+# update cryptography >=41.0.0: CVE-2023-2650. This comes from the base image
+# update tornado >= 6.3.2: CVE-2023-28370. This comes from the base image
+# update pyOpenSSL 23.2.0 so cryptography can be updated
+# pin mmengine to avoid MultiModalPredictor import crashing
+# pin yapf to resolve importlib-metadata conflict with sagemaker==2.150.0
+RUN pip install --no-cache-dir -U --trusted-host pypi.org --trusted-host files.pythonhosted.org pip \
+ && pip install --no-cache-dir -U setuptools wheel \
+ && pip install --no-cache-dir -U numpy numba \
+ && pip install --no-cache-dir -U autogluon==${AUTOGLUON_VERSION} \
+ && mim install -q mmcv \
```

Comment:
Shall we install 'pymupdf` as well as it is moved to extra dependency at [here](https://github.com/autogluon/autogluon/blob/master/multimodal/setup.py#L72C10-L72C17)

Reply:
Good point. Let me add that